setup.sh

```diff
@@ -9,7 +10,7 @@ apt-add-repository -y ppa:juju/stable
 apt-get update -qq
 
 apt-get -qy install juju-2.0
-apt-get -qy install byobu vim charm-tools openssh-client
+apt-get -qy install byobu vim charm-tools openssh-client sudo
```

Comment:
Minor nit. We should create a list of packages to install instead of separate calls to apt-get.